libs/form/src/next/input-field/stories/vl-input-field.stories-arg.ts

```diff
@@ -103,22 +103,52 @@ export const inputFieldArgTypes: ArgTypes<typeof inputFieldArgs> = {
     max: {
         name: 'max',
         description: 'De maximum waarde die je kan ingeven.',
-        control: { type: 'number' },
+        control: { type: CONTROLS.NUMBER },
         table: {
             type: { summary: TYPES.NUMBER },
             category: CATEGORIES.ATTRIBUTES,
             defaultValue: { summary: inputFieldArgs.max },
         },
     },
+    minExclusive: {
+        name: 'min-exclusive',
```

Comment:
Dit is voor mij zeker geen must om te veranderen, maar gezien `min` en `max` ook al een shorthand is voor `minimum` en `maximum`, zou het dan niet ok zijn om ook `min-excl` te gebruiken voor de attribuut name? (en gelijkaardig dan voor max; `max-excl`)

Ok voor mij om het zo te houden hoor.

Reply:
Besproken en gekozen om `exclusive` voluit te schrijven.